api/app/routers/hfi_calc.py

```diff
@@ -62,6 +63,72 @@ def extract_selected_stations(request: HFIResultRequest) -> List[int]:
     return stations_codes
 
 
+@router.post("/load", response_model=HFIResultResponse)
+async def load_hfi_result(request: HFILoadResultRequest,
+                          response: Response,
+                          token=Depends(authentication_required)):
+    """ Given a fire centre id (and optionally a start date), load the most recent HFIResultRequst.
```

Comment:
HFIResultRequst -> HFIResultRequest

Reply:
sometimes @conbrad eases my workload of pointing out all of Sy's spellign misteaks. I like it!

Reply:
I cnt evn c thu prblm.